packages/react/src/components/DataGrid/index.stories.tsx

```diff
@@ -84,17 +87,6 @@ export const Loading = () => {
 };
 
 export const ClientSideWithoutPagination = () => {
-  const database = useMemo(
-    () =>
-      Array.from(Array(23)).map(() => ({
-        id: faker.string.uuid(),
-        firstName: faker.person.firstName(),
-        lastName: faker.person.lastName(),
-        email: faker.internet.email(),
-        address: faker.location.streetAddress(),
-      })),
-    [],
```

Comment:
Maybe we could simply use a seed to always get same results ?

Reply:
I would work only if the amount of stories does not change 🤔

Reply:
Not quit sure to understand why ?